dotnet/src/InternalUtilities/connectors/AI/FunctionCalling/FunctionCallsProcessor.cs

```diff
@@ -129,6 +130,7 @@ public FunctionCallsProcessor(ILogger? logger = null)
     /// <returns>Last chat history message if function invocation filter requested processing termination, otherwise null.</returns>
     public async Task<ChatMessageContent?> ProcessFunctionCallsAsync(
```

Comment:
Should this be public (is already a internal class and utility) ? 

```suggestion
    internal async Task<ChatMessageContent?> ProcessFunctionCallsAsync(
```